themes/janeswalk/js/components/pages/WalkPage/WalkHeader.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+//TODO: Define as a path in a common place to share with other components
```

Comment:
It's more clear with every block and page we write that we'll need this. I'm thinking the best spot might be to remove them from the concrete5 and PHP autoloader's domain completely. We could bundle these in a custom, private NPM packages, e.g. you could then say `import {dateFormatted} from 'janeswalk/utils/ItineraryUtils'.